.github/workflows/pr-quality-report.yml

```
@@ -0,0 +1,275 @@
+          mkdir -p coverage
+        env:
+          NODE_OPTIONS: '--experimental-vm-modules'
+
+      - name: Generate lint report
+        run: |
+          npm run lint -- --format=json > lint-report.json || true
+          npm run lint > lint-report.txt 2>&1 || true
+
+      - name: Check lint ratchet
+        id: lint-ratchet
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:14:46: Double quote to prevent globbing and word splitting

(shellcheck)